src/lib/core/WeaveDMConfig.h

```diff
@@ -467,23 +467,36 @@
 #endif
 
 /**
- *  @def WDM_PUBLISHER_ENABLE_CUSTOM_COMMANDS
+ *  @def WDM_PUBLISHER_ENABLE_CUSTOM_COMMAND_HANDLER
  *
  *  @brief
  *    Enable (1) or disable (0) custom command handler support
  *    in Weave Data Management Next profile. This feature is
  *    optional and could be disabled, for example, to save code space.
  *
  */
-#ifndef WDM_PUBLISHER_ENABLE_CUSTOM_COMMANDS
-#define WDM_PUBLISHER_ENABLE_CUSTOM_COMMANDS 1
+#ifndef WDM_PUBLISHER_ENABLE_CUSTOM_COMMAND_HANDLER
+#define WDM_PUBLISHER_ENABLE_CUSTOM_COMMAND_HANDLER 1
 #endif
 
-#if WDM_PUBLISHER_ENABLE_CUSTOM_COMMANDS
-#ifndef WDM_MAX_NUM_COMMAND_OBJECTS
-#define WDM_MAX_NUM_COMMAND_OBJECTS (WDM_MAX_NUM_SUBSCRIPTION_HANDLERS)
-#endif // WDM_MAX_NUM_COMMAND_OBJECTS
-#endif // WDM_PUBLISHER_ENABLE_CUSTOM_COMMANDS
+#if WDM_PUBLISHER_ENABLE_CUSTOM_COMMAND_HANDLER
+#ifndef WDM_MAX_NUM_COMMAND_HANDLER_OBJECTS
+#define WDM_MAX_NUM_COMMAND_HANDLER_OBJECTS (WDM_MAX_NUM_SUBSCRIPTION_HANDLERS)
+#endif // WDM_MAX_NUM_COMMAND_HANDLER_OBJECTS
+#endif // WDM_PUBLISHER_ENABLE_CUSTOM_COMMAND_HANDLER
+
+/**
+ *  @def WEAVE_CONFIG_ENABLE_WDM_CUSTOM_COMMAND_SENDER
+ *
+ *  @brief
+ *    Enable (1) or disable (0) command sender support
+ *    in Weave Data Management Next profile. This feature is
+ *    optional and could be disabled.
+ *
+ */
+#ifndef WEAVE_CONFIG_ENABLE_WDM_CUSTOM_COMMAND_SENDER
+#define WEAVE_CONFIG_ENABLE_WDM_CUSTOM_COMMAND_SENDER 0
```

Comment:
From previous PR (@jaylogue):

Does this need to be off by default? I presume it won't cause any extra code to be linked if no one references it.

Reply:
@mrjerryjohns Is this weave config even necessary?

Reply:
Agreed, I think it's safe to enable it by default given the natural dead-stripping that'll happen if there are no references to it.